js/displayProducts.js

```diff
@@ -0,0 +1,192 @@
+                        ${rating(5)}
+                        <p class="product-row__desc d-md-none d-lg-block">${featuredProducts[i].desc}</p>
+                        ${prices(featuredProducts[i].curPrice)}
+                        <div class="btn__wrapper">
+                          <button class="btn-order add-btn" data-category="featuredProducts" data-id=${featuredProducts[i].id} >MUA NGAY</button>
+                          <button class="btn-rounded"> <i class="fa-solid fa-magnifying-glass"></i></button>
+                          <button class="btn-rounded"> <i class="fa-solid fa-heart"></i></button>
+                        </div>
+                      </div>
+                    </div>
+                  </div>
+                </div>
+              </div>`
+    };
+
+    productList.innerHTML = list;
+  }
+};
+
+const displayBestSelling = (bestSellingProducts) => {
+  const productList = document.querySelector(".bestproduct__container");
+
+  if(productList) {
+    let list = "";
+
+    for(let i = 0; i < bestSellingProducts.length; i++){
+      list += `<div class="bestproduct__wrapper">
+                <div class="d-flex product-sm"> 
+                  <div class="col-md-5 m-2 img__container"><a> <img src=${bestSellingProducts[i].imgSrc}></a>
+                    <div class="product-hover"> 
+                      <button class="btn-rounded"> <i class="fa-solid fa-magnifying-glass"></i></button>
+                    </div>
+                  </div>
+                  <div class="col-md-7 ms-2 d-flex flex-column justify-content-center">
+                    <p class="mb-1">${bestSellingProducts[i].title}</p>
+                    <div class="d-flex mb-1">
+                      ${rating(5)}
+                    </div>
+                    ${prices(bestSellingProducts[i].curPrice)}
+                  </div>
+                </div>
+              </div>`
+    };
+
+    productList.innerHTML = list;
+  }
+};
+
+const displayDiscounts = (discountProducts) => {
+  const productList = document.getElementById('discount-products');
+
+  if(productList) {
+    let list = "";
+
+    for(let i = 0; i < discountProducts.length; i++){
+      list += `<div class="col-md-4">
+                ${productLarge(discountProducts[i], "discountProducts")}
+              </div>`
+    };
+
+    productList.innerHTML = list;
+  }
+};
+
+const displayNewProducts = (newProducts) => {
+  const productList = document.getElementById('new-products');
+
+  if(productList) {
+    let list = "";
+
+    for(let i = 0; i < newProducts.length; i++){
+      list += `<div class="col-md-3">
+                ${productLarge(newProducts[i], "newProducts")}
+            </div>`
+    };
+
+    productList.innerHTML = list;
+  }
+};
+
+const rating = (num) => {
+  let stars = `
+      <div class="star-rating">
+  `;
+
+  for(let i = 0; i < num; i++) {
+    stars += `<i class="fa-solid fa-star"></i>`;
+  }
+
+  stars += `
+    </div>
+  `
+
+  return stars
```

Comment:
Trong trường hợp num = 3 thì sao, nó chỉ hiện mỗi 3 sao
```suggestion
  const stars = [...Array(5)].map((_, index) => index < num ? 'fa-solid fa-star' : 'fa-solid fas-star')
  
  return `
      <div class="star-rating">
          {stars.map(star) => `<i class="${star}"></i>`}
      </div>
  `
```